chatbot.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""ChatBot.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1r_F0WtD5mllbnGUuIqevrml7QrFCKads
+"""
+
```

Comment:
Installing nltk for natural language processing. 
Creating pairs for conversations
Taking user input